.pre-commit-config.yaml

```diff
@@ -1,41 +1,53 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
It works perfectly fine without the indentation.

Reply:
"it works" applies to ALL linting checks here. We are trying to keep it tidy - minimize changes, keep things consistent -- not just working.   The indented list is a pretty common formatting rule - intellij does it by default, as well as some guidelines like this one https://ref.coddy.tech/yaml/yaml-formatting-guidelines